### PR TITLE
Draft site redesign

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

File: _src/images/logos/optimism-uniswap.svg [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (alexroan) [inline _src/images/logos/optimism-uniswap.svg:1]: Old Uniswap logo instead of Optimism?

**Review by alexroan — CHANGES_REQUESTED** [review]

File: TEMPLATE.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ metadata:
   image:
     0: "insert image here"
 ---
-*This doc is meant to be used as a template for creating a tutorial doc. Contributors should emulate the structure of this document when creating new pages. The content of this doc comes from beginners-tutorial.md.*
+*This doc is meant to be used as a template for creating a tutorial doc. Contributors should emulate the structure of this document when creating new pages. The content of this doc comes from the Getting Started Guides.*
```
Comment (StephenFluin) [inline TEMPLATE.md:13]: Add a link maybe?

File: _src/docs.css [hunk 3]
```diff
@@ -478,6 +548,155 @@ hr {
   font-display: swap;
 }
 
+/** Custom Chain Cards **/
```
Comment (StephenFluin) [inline _src/docs.css:551]: If this CSS is specific to only the home page, can we scope it to that page only (move the CSS to that page?)

File: _src/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (StephenFluin) [inline _src/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg:1]: Can we come up with a more descriptive folder name than `misc` for these? Maybe which section of the site they will live in?

File: docs/home.html [hunk 5]
```diff
@@ -0,0 +1,293 @@
+---
+layout: home.liquid
+title: 'Chainlink Documentation'
```
Comment (StephenFluin) [inline docs/home.html:3]: Should we get rid of the top bar with `chainlink documentation` and the feedback bar? Feels duplicative with the header.

File: docs/Introduction/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+section: gettingStarted
+title: "Getting Started with Chainlink"
+date: Last Modified
+permalink: /getting-started/
+---
+
+<div>
+  <div class="markdown-body">
+    <div class="cl-section-header">
+      <h2>Start your journey with Chainlink</h2>
```
Comment (StephenFluin) [inline docs/Introduction/getting-started.md:12]: On page this feels visually duplicative with the title "Getting Started with Chainlink". should we just pick one?

File: docs/Introduction/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+                <h3>Join the Community</h3>
+            </div>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+Running your own Chainlink node? See the [Node Operator's Guide](/chainlink-nodes/) to learn about starting and maintaining Chainlink Nodes.
```
Comment (StephenFluin) [inline docs/Introduction/getting-started.md:169]: This looks like it only applies to folks who are already running a node, what about people who want to run nodes?

File: docs/Introduction/getting-started/conceptual-overview.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+
+A valuable feature of smart contracts is that they can store and manage on-chain assets (like [ETH or ERC20 tokens](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/)), just like you can with an Ethereum wallet. Because they have an on-chain address like a wallet, they can do everything any other address can. This opens the door for programming automated actions when receiving and transferring assets.
+
+Smart contracts can connect to real-world market prices of assets to produce powerful applications. Being able to securely connect smart contracts with off-chain data and services is what makes them *hybrid* smart contracts. This is done using oracles.
+
+## What language is a smart contract written in?
+
+The most popular language for writing smart contracts on Ethereum is [Solidity](https://docs.soliditylang.org/en/v0.8.7/). It was created by the Ethereum Foundation specifically for smart contract development and is constantly being updated.
+
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/conceptual-overview.md:49]: I feel like we need to at least mention Rust/Solana/others here.

File: docs/Introduction/getting-started/conceptual-overview.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+### Solidity versions
+
+The first thing that every Solidity file must have is the Solidity version definition. The `HelloWorld.sol` contract uses version `0.8.7`, which is defined in the contract as `pragma solidity 0.8.7;`
+
+You can see the latest versions of the Solidity compiler [here](https://github.com/ethereum/solc-bin/blob/gh-pages/bin/list.txt/?target=_blank). You might also notice smart contracts that are compatible with a range of versions.
+
+```solidity
+pragma solidity >=0.7.0 <0.9.0;
+```
+This means that the code is written for Solidity version 0.7.0, or a newer version of the language up to, but not including version 0.9.0. In short, `pragma` is used to instruct the compiler as how to treat the code.
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/conceptual-overview.md:74]: This isn't quite right, it's actually used to CHOOSE the compiler.

File: docs/Introduction/getting-started/deploy-your-first-contract.md [hunk 10]
```diff
@@ -0,0 +1,120 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+date: Last Modified
+title: "Deploy Your First Smart Contract"
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/deploy-your-first-contract.md:5]: We use deploy multiple ways in our docs. We "deploy to remix" as well in earlier steps. Should we rename all of our remix buttons to something else?

File: docs/Introduction/getting-started/conceptual-overview.md [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+
+## What are oracles?
+
+**Oracles** provide a bridge between the real-world and on-chain smart contracts, by being a source of data that smart contracts can rely on, and act upon.
+
+Oracles play an extremely important role in facilitating the full potential of smart contract utility. Without a reliable connection to real-world conditions, smart contracts are unable to effectively serve the real-world.
+
+## How do smart contracts use oracles?
+
+Oracles are most popularly used with [*Data Feeds*](../using-chainlink-reference-contracts/). DeFi platforms like [AAVE](https://aave.com/) and [Synthetix](https://www.synthetix.io/) use Chainlink data feed oracles to obtain accurate real-time asset prices in their smart contracts.
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/conceptual-overview.md:153]: I think we need to mention all of our products, or talk more generally about Oracles giving you access to data and capabilities.

File: docs/Introduction/getting-started/deploy-your-first-contract.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+  description: "Deploy and run your first smart contract in an interactive web development environment."
+  image:
+    0: "/files/1a63254-link.png"
+---
+
+You can write your first smart contract and run in your browser without any knowledge about Ethereum or blockchains. This guide shows you how easy it is to develop smart contracts using the [Solidity language](https://soliditylang.org/), a [MetaMask wallet](https://metamask.io) and the [Remix Development Environment](https://remix.ethereum.org/). You can use all of these tools in your browser for free with no signup required.
+
+> 📘 Already know Solidity and have MetaMask configured?
+>
+> If you are already familiar with how to deploy Smart Contracts and use MetaMask, you can skip to the [<!--TODO: Add link-->]() guide to learn how to connect your smart contracts to Chainlink data feeds.
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/deploy-your-first-contract.md:19]: Todo here still.

File: _data/navigation.js [hunk 13]
```diff
@@ -1,230 +1,80 @@
...
-            url: '/docs/performing-system-maintenance/',
-            title: 'Performing System Maintenance',
+            title: 'Consuming Data Feeds',
+            url: '/docs/consuming-data-feeds/',
           },
           {
-            url: '/docs/connecting-to-a-remote-database/',
-            title: 'Connecting to a Remote Database',
+            title: 'Get Random Numbers',
+            url: '/docs/intermediates-tutorial/',
```
Comment (StephenFluin) [inline _data/navigation.js:21]: It looks like there's no Any API 'getting started'. I think that's important as this is the #1 product developers actually want to use for things like hackathons. How can we help thoes users?

File: docs/Introduction/getting-started/fund-your-contract.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+date: Last Modified
+title: "Fund Your Contracts"
+permalink: "docs/fund-your-contract/"
+---
+
+Some smart contracts require funding at their addresses so they can operate without you having to call functions manually and pay for the transactions through MetaMask. This guide explains how to fund Solidity contracts with LINK or ETH using the Remix IDE.
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/fund-your-contract.md:9]: You only ever fund like this if you use VRF, and that's going away. Should we just leave this page off of the getting started?

File: docs/Introduction/getting-started/other-tutorials.md [hunk 15]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: nodes.liquid
-section: smartContract
+section: gettingStarted
 date: Last Modified
 title: "Learning Resources"
 permalink: "docs/other-tutorials/"
 ---
```
Comment (StephenFluin) [inline docs/Introduction/getting-started/other-tutorials.md:7]: It looks like "Smart Contract Learning Materials" is an empty header? Is that a design problem?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Can you also make sure the images are run through an optimizer? I can help with that if needed.

File: docs/Introduction/getting-started/other-tutorials.md (hunk 15 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/other-tutorials.md:7]: Removed and updated the sub-headers.

File: docs/Introduction/getting-started/deploy-your-first-contract.md (hunk 12 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/deploy-your-first-contract.md]: Completed

File: docs/Introduction/getting-started/conceptual-overview.md (hunk 9 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/conceptual-overview.md]: Updated this sentence to be more accurate.

File: docs/Introduction/getting-started.md (hunk 7 above)
Comment (dwightjl) [reply docs/Introduction/getting-started.md]: Restored the original text. "Need to run your own Chainlink Node?"

File: TEMPLATE.md (hunk 2 above)
Comment (dwightjl) [reply TEMPLATE.md]: Added a link to the folder in GitHub.

File: _src/images/logos/optimism-uniswap.svg (hunk 1 above)
Comment (dwightjl) [reply _src/images/logos/optimism-uniswap.svg]: Updated with the new logo.

File: _src/docs.css (hunk 3 above)
Comment (dwightjl) [reply _src/docs.css:553]: Will do this in a followup.

File: docs/home.html (hunk 5 above)
Comment (dwightjl) [reply docs/home.html:3]: Removed.

File: docs/Introduction/getting-started.md (hunk 6 above)
Comment (dwightjl) [reply docs/Introduction/getting-started.md]: Moved to the title.

File: docs/Introduction/getting-started/deploy-your-first-contract.md (hunk 10 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/deploy-your-first-contract.md:5]: Standardized the buttons to say "Open in Remix" except in cases where custom button action text is used.

File: _data/navigation.js (hunk 13 above)
Comment (dwightjl) [reply _data/navigation.js:21]: Restored from the main docs to Getting Started for now.

File: docs/Introduction/getting-started/fund-your-contract.md (hunk 14 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/fund-your-contract.md]: Moved to the Ethereum Chains docs in the Developer Resources nav section.

File: _src/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg (hunk 4 above)
Comment (dwightjl) [reply _src/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg]: Changed to `card-icons` because these icons are used exclusively for cards.

**Review by PatrickAlphaC — COMMENTED** [review]
Should the chainlink architecture page _only_ be on the ethereum chains page?

**Comment by PatrickAlphaC** [comment]
Made a few comments in my review, let me know what you think.

**Comment by PatrickAlphaC** [comment]
The rest looks pretty good. I love the new landing page. Overall, I'd love the getting started to be one less click. I think ideally it drops us into the conceptual overview.

File: docs/Introduction/getting-started.md [hunk 16]
```diff
@@ -0,0 +1,193 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+title: "Start your journey with Chainlink"
+date: Last Modified
+permalink: /getting-started/
+---
+
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Introduction/getting-started.md]: Good point. Landing pages are not optimal, so why create another one? I moved relevant info to the EVM Chains landing page, and made the Overview the new landing page. One less click.

File: TEMPLATE.md [hunk 17]
```diff
@@ -10,13 +10,13 @@ metadata:
   image:
     0: "insert image here"
 ---
-*This doc is meant to be used as a template for creating a tutorial doc. Contributors should emulate the structure of this document when creating new pages. The content of this doc comes from beginners-tutorial.md.*
+*This doc is meant to be used as a template for creating a tutorial doc. Contributors should emulate the structure of this document when creating new pages. The content of this doc comes from the [Getting Started Guides](https://github.com/smartcontractkit/documentation/tree/main/docs/Introduction/getting-started/).*
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply TEMPLATE.md:13]: Guides describes the getting started content. It's a set of guides, so it's important to convey expectations to the reader. Are you thinking about Getting Started in a different way? I can change my approach here if you think it's more clear to refer to Getting Started content differently.

File: docs/Introduction/getting-started/conceptual-overview.md (hunk 11 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/conceptual-overview.md:161]: Added all of our products to this section with descriptions.

File: docs/Introduction/getting-started/conceptual-overview.md (hunk 8 above)
Comment (dwightjl) [reply docs/Introduction/getting-started/conceptual-overview.md:49]: I added the non-EVM chains to this section with their supported languages; links to their language documentation.

File: TEMPLATE.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (PatrickAlphaC) [reply TEMPLATE.md:13]: Nevermind, I actually like what you have.

**Review by PatrickAlphaC — APPROVED** [review]

**Review by ritika-b — APPROVED** [review]

**Comment by dwightjl** [comment]
Rebased to main.

**Pull request merged**